frontend/test/pytest/test_template.py

```diff
@@ -171,6 +171,7 @@ def basis_state_preparation(basis_state):
     assert np.allclose(interpreted_fn(params), jitted_fn(params))
 
 
+@pytest.mark.skip(reason="GlobalPhase not implemented in QIR")
```

Comment:
This should be removed (my might have missed it in the GP pr?)
```suggestion
@pytest.mark.skip(reason="GlobalPhase not implemented in QIR")
```

Reply:
https://github.com/PennyLaneAI/catalyst/pull/571/commits/f434364847fe5b3563d99d2c45b3203415c318e9